go.mod

```diff
@@ -23,34 +23,54 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
+	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a // indirect
```

Comment:
There should be no need to update go.mod because of the change in Makefile(?)

Reply:
my bad, some how I ran gomod tidy here

Reply:
> my bad, some how I ran gomod tidy here

No problem. `go mod tidy` would make sense, but perhaps in a separate PR 😊